client/settings/payment-settings/promotional-banner/get-promotional-banner-type.js

```diff
@@ -35,7 +36,12 @@ export const getPromotionalBannerType = (
 
 	if ( oauthConnected === false ) {
 		return RECONNECT_BANNER;
-	} else if ( isUpeEnabled && ! hasBNPLEnabled ) {
+	} else if (
+		isUpeEnabled &&
+		! hasBNPLEnabled &&
+		// eslint-disable-next-line camelcase
+		wc_stripe_settings_params.plugin_version >= '9.7.0'
```

Comment:
Will `wc_stripe_settings_params` always be defined and an object?

Reply:
I think we need to make this a bit more robust, as the `>=` comparison will break when we hit `10.0.0`, as that returns false for this comparison.

Maybe we can add the following checks to make extra-sure that we have a numeric major.minor that's greater than or equal to 9.7?
```suggestion
		wc_stripe_settings_params?.plugin_version &&
		parseFloat( String( wc_stripe_settings_params.plugin_version ).split( '.' ).slice( 0, 2 ).join( '.' ) ) >= 9.7
```